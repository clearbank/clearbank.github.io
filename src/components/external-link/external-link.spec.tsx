import React from 'react';
import { render } from '@testing-library/react';
import ExternalLink from './external-link';

test('renders external link with icon', () => {
  const { getByText } = render(<ExternalLink href="https://gov.uk">Example text</ExternalLink>);
  const linkElement = getByText(/Example text/i);
  expect(linkElement).toBeInTheDocument();
  expect(linkElement).toHaveAttribute('href', 'https://gov.uk');
  expect(linkElement).toHaveAttribute('target', '_blank');
});