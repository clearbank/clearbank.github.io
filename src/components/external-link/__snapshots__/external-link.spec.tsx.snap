// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders external link with icon 1`] = `
    <a href="https://gov.uk" target="_blank" rel="noopener noreferrer"> Example text <span class="sc-bdfBwQ" /></a>
`;
